packages/api-explorer/package-lock.json

```diff
@@ -11320,30 +11317,20 @@
       },
       "dependencies": {
         "ajv": {
-          "version": "6.10.0",
```

Comment:
@dok Did you mean to downgrade ajv here?

Reply:
I did not intend to downgrade. The update to `package-lock.json` was a result of running `npx lerna bootstrap`.

Reply:
K cool, just checking. Looks like 5.5.2 falls in line with what the `react-jsonschema-form` dependency is requesting anyways so 👍